### DISCUSSION TITLE
part-ii/2019-47/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#262] [Question comments]
Part II 2019 Differential Geometry: Paper 3, Section II, H 

https://questions.tripos.org/part-ii/2019-47/

**Comment by stevezshen** [comment]
<details>
<summary>Spoiler warning</summary>

(a) (ii) Write down/derive the equations for geodesics in terms of the coordinates $(u(s),v(s))$, (ES3 S4) then the claims follow immediately. 

(b)

(i) $g'(a) = g'(b) = 0$ is necessary and sufficient. (I think)

(ii) $r\to 0$ as $v\to a$ or $v\to b$, so by inspection in Clairaut's relation, for any given geodesic, its radius has a positive lower bound.
</details>
